libnavui-base/src/main/java/com/mapbox/navigation/ui/base/view/MapboxExtendableButton.kt

```diff
@@ -21,8 +21,8 @@ import com.mapbox.navigation.ui.utils.internal.extensions.measureTextWidth
 /**
  * Button with an icon and expand behaviour.
  */
-@ExperimentalPreviewMapboxNavigationAPI
 @UiThread
+@ExperimentalPreviewMapboxNavigationAPI
```

Comment:
Instead of marking this class experimental, why not just mark`fun updateStyle(@StyleRes style: Int)` `@Deprecated`?

Reply:
It sounds a bit funny to release a product in GA with `@deprecated` API. I have rather removed the `Experimental` annotation from the class and only marked that particular API as `Experimental`.

Reply:
Or... since the class is still experimental, and to make it stable we need to remove `updateStyle` method,  just delete the method.

Reply:
Done.